docs/src/ArtifactCalling.md

```diff
@@ -0,0 +1,20 @@
+# Artifact Calling
+
+## Using Artifacts
+Calling the artifact needs to be done in the `CloudMicrophysics` source code so that the 
+    `Artifact.toml` file can be seen. The file containing the list of artifacts is
+    `ArtifactCalling.jl`. To call an artifact in a certain file outside of the `src` folder,
+    simply call the calling function just as you would other functions in the source code.
+    For example:
+    
+```
+import CloudMicrophysics as CM
+
+data_file_name = "in05_17_aida.edf"
+CM.ArtifactCalling.AIDA_ice_nucleation(data_file_name)
+```
+This will output the filepath of the data file named `in05_17_aida.edf` which is stored in
+the Caltech box.
```

Comment:
To be pedantic, `in05_17_aida.edf` will be stored on your computer at this point.
I'd say something about "this will automatically download the file from the Caltech box if not already available"